definitions/npm/backbone_v1.3.x/flow_>=v0.25.x/backbone_v1.3.x.js.flow

```diff
@@ -0,0 +1,68 @@
+// @flow
+import typeof Underscore from 'underscore';
+
+declare module 'backbone' {
+   declare var $: any;
```

Comment:
Using `any` is likely the best way for now since the import you made above won't work.

If you leave a comment `// $FlowFixMe` above the `any` we will return to it.